bundle/src/test/java/com/adobe/acs/commons/redirects/filter/RedirectFilterTest.java

```diff
@@ -1006,4 +1044,22 @@ public void testSelectorsDisabled() throws Exception {
         assertEquals("/content/geometrixx/en/page1.html", navigate("/content/geometrixx/en/one.mobile.html").getHeader("Location"));
         assertEquals("/content/geometrixx/en/page1.html", navigate("/content/geometrixx/en/one.desktop.html").getHeader("Location"));
     }
+
+    @Test
+    public void testEvaluateURI() throws Exception {

```

Comment:
please add a few more test cases:

1. a simple (non-regex) request URI rule, e.g. `/content/geometrixx/en/page.html/suffix.html`
2. a more complex URI with selectors and a suffix, e.g.  `/content/geometrixx/en/page.mobile.html/suffix.html`
3. a regex capturing different parts of the URI, e.g. `/content/geometrixx/en/(\\w+).(mobile|desktop).html/suffix-(\\d+).html`
4. Since query string is not a part of the request URI, it should not affect evaluation. Given a rule `/content/geometrixx/en/page.html/suffix.html`, it should match `/content/geometrixx/en/page.html/suffix.html?a=b&c=d` and `/content/geometrixx/en/page.html/suffix.html#here`. Please include a junit for this as well.